components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -80,6 +82,11 @@
   /** TIFF tiles must be of a height and width divisible by 16. */
   private static final int TILE_GRANULARITY = 16;
 
+  /** Automatic tile writing off by default unless set in MetadataOptions using DEFAULT_TILE_WRITING_KEY */
+  public static final String DEFAULT_TILE_WRITING_KEY = "bioformats.tiff.defaultTileWriting";
```

Comment:
Do we have a policy for key naming?  This looks a little wordy.  `bioformats` is redundant since it's implicitly part of bioformats, as is `Writing`.  Would something like `tiff.tile = true` or `tiff.layout = tile|strip|scanline` be a bit more concise and to the point?  Also, why encode `default` in the name; isn't it implicitly a default just by virtue of being a default value?